src/jenkins.py

```diff
@@ -3,6 +3,8 @@
 
 """Functions to operate Jenkins."""
 
+# pylint: disable=too-many-lines
```

Comment:
Perhaps this needs to be split into a few submodules now?

Reply:
Yes, but it seems the me the PR is already too bit. I plan to do that in a follow up PR along with some more refactoring